### PR TITLE
add CORS resolution in WP

### LINKED ISSUE #4270: Platform Considerations Doc Update CORS implementation in WP & Drupal
Re: [Platform Considerations](https://pantheon.io/docs/platform-considerations/)

Priority: Low

## Issue Description:
Add details on how to implement CORS  in WP & Drupal

## Suggested Resolution
To further help customers that has CORS issue, it is better to point them on the next steps like what plugin to use https://wordpress.org/plugins/wp-cors/ or filter to use https://wordpress.stackexchange.com/questions/215053/cannot-load-admin-ajax-php-no-access-control-allow-origin

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes #4270 

## Effect
PR includes the following changes:
- Add possible solutions in WP

## Remaining Work
- [ ] Add possible solutions in Drupal

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

File: source/_docs/platform-considerations.md [hunk 1]
```diff
@@ -164,6 +164,17 @@ For sites that need to provide services with Cross-Origin Resource Sharing (COR
 
 Sites that consume services using CORS, such as Amazon S3 CORS, do work on Pantheon.
 
+For WordPress users, they can use the [WP-CORS plugin](https://wordpress.org/plugins/wp-cors/){.external} or add this in their theme's `function.php`:
+
+```
+add_filter('allowed_http_origins', 'pantheon_allowed_origins');
+
+function pantheon_allowed_origins($urls) {
+    $urls[] = 'https://www.example.com';
```
Comment (alexfornuto) [inline source/_docs/platform-considerations.md]: @carl-alberto what would `example.com` be replaced with in this example?

Comment (carl-alberto) [reply source/_docs/platform-considerations.md]: @alexfornuto These will be the urls allowed for cross-domain requests, please note that protocol(http/https) and www/non www urls are relevant here. Multiple values is also possible, eg.

`$urls[] = array( 'https://www.example.com', 'http://www.example.com', 'https://example.com', 'http://example.com'  ) ;`

File: source/_docs/platform-considerations.md [hunk 2]
```diff
@@ -160,10 +160,21 @@ PHP short tags (`<? ... ?>`) are not supported on Pantheon. The [PHP Manual](htt
 
 ## CORS
 
-For sites that need to provide services with Cross-Origin Resource Sharing (CORS), adding the proper header should enable the resource. See  [https://enable-cors.org/server\_php.html](https://enable-cors.org/server_php.html)
+For sites that need to provide services with Cross-Origin Resource Sharing (CORS), adding the proper header should enable the resource. See  [https://enable-cors.org/server_php.html](https://enable-cors.org/server_php.html)
 
 Sites that consume services using CORS, such as Amazon S3 CORS, do work on Pantheon.
 
+For WordPress users, you can use the [WP-CORS plugin](https://wordpress.org/plugins/wp-cors/){.external}, or add the following to the theme's `function.php`:
```
Comment (alexfornuto) [inline source/_docs/platform-considerations.md]: Is "the theme's" referring to whatever the current active theme is, or is it part of the WP-CORS plugin?

Comment (carl-alberto) [reply source/_docs/platform-considerations.md]: That would be the current active theme's function.php

**Pull request merged**